beps/0001-notifications-system/README.md

```diff
@@ -132,6 +136,7 @@ If a scope is provided for a notification, it will either:
 - Update existing notification in that scope and origin so that it's unread considered as a new notification
 
 The idea of scope is to be able to reuse notifications about same information for example specific failing CI build.
+Payload of notifications without scope is meant to be immutable.
```

Comment:
This is unrelated to this PR, just bringing it up to encourage some thing around it: I don't really feel that "scope" name conveys the intent and behavior of this field, and it would be best to rename it to something else

Reply:
Honestly, I would remove it completely. The use-case with CI can be handled via filtering "the latest of a topic" message.

Reply:
I wouldn't remove it. It's very many times use case that you want to re-use the same notification. For example in my Q&A plugin user creates a question. 2 other users replies to it so I don't want to send 2 notifications about having new replies for the same question.

Reply:
What comes to naming, I really can't think better one 😆 If you have ideas, please comment!

Reply:
I am not eager to remove it.

Anyway, the `scope` seems to be kind of `key` which is used to repeatedly find&edit the same message.
Technically, the recent `id` should be enough.

Reply:
I see use cases for this. It's optional anyway so is there some harm in keeping it?

Reply:
The benefit of having something like scope over ID is that you can avoid the need for plugins to store state about notifications, as they can drive the scope from their own state